hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -382,15 +395,19 @@ async def _api_request(
         signature to the request.
         :returns A response in json format.
         """
+
+        data = json.dumps(data)
+
```

Comment:
This might not be required. Please double check if this is being done already by the RESTAssistant when executing POST requests.

Reply:
It's required. Without it the requests were being rejected.